Sources/BrowserServicesKit/Autofill/AutofillUserScript.swift

```diff
@@ -54,6 +56,11 @@ public class AutofillUserScript: NSObject, UserScript, UserScriptMessageEncrypti
         case checkCredentialsProviderStatus
         
         case sendJSPixel
+
+        case setIncontextSignupPermanentlyDismissedAt
```

Comment:
+1, but will punt for now as it's not an easy fix -- in the autofill code we use `Incontext` and in the native code we use `InContext`, but because this is a message being triggered by the autofill code, it doesn't match the rest of the codebase.